src/Performance/Minibench.purs

```diff
@@ -22,6 +22,10 @@ import Prelude hiding (min,max)
 -- | A wrapper around the Node `process.hrtime()` function.
 foreign import hrTime :: forall eff. EffFn1 eff (Array Int) (Array Int)
 
+-- | Force garbage collection.
+-- | Requires node to be run with the --force-gc flag.
```

Comment:
Maybe this comment should be moved to the top level?

Reply:
Tracking this in #21